packages/@pollyjs/core/src/polly.js

```diff
@@ -119,44 +121,32 @@ export default class Polly {
       this.mode !== MODES.STOPPED
     );
 
-    const { _container: container } = this;
-
     this.config = mergeOptions(DefaultConfig, this.config, config);
 
-    // Handle Adapters
-    this.config.adapters.forEach(adapter => {
-      let adapterName = adapter;
+    /* Handle Adapters */
 
-      if (isArray(adapterName)) {
-        const [name, AdapterType] = adapterName;
+    // Disconnect from all current adapters
+    this.disconnect();
```

Comment:
Are there tests asserting disconnect on configure (quick grep didn't find anything)?  Would be useful if there isn't.

Reply:
Done.